### DISCUSSION TITLE
part-ii/2017-7/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#60] [Question comments]
Part II 2017 Algebraic Topology: Paper 4, Section II, I 

https://questions.tripos.org/part-ii/2017-7/

**Comment by eha1234** [comment]
For (c), I want to say for $n$ odd it's orientation reversing so acts like -1, but this seems a bit sketchy. Also hints for (d)?

**Reply by eha1234** [reply]
<details><summary>(d)</summary>

Hint from supervisor: Flow in direction of $f(x)$ to get that id is homotopic to antipodal map, contradiction. 
</details>

**Reply by artie2000** [reply]
For (c), you want to use that antipodal is homotopic to identity.

Alternatively, use Lefschetz fixed point theorem.

**Comment by artie2000** [comment]
<details>
<summary>(a)</summary>

Map $x\to (-x_2,x_1,\dots,-x_{2k},x_{2k-1})$ (in $\mathbb{R}^{2k}$).

</details>
<details>
<summary>(b)</summary>

Embed an $(n+1)$-simplex symmetrically in $\mathbb{R}^{n+1}$ (for example by viewing $\mathbb{R}^{n+1}$ as the plane $\{\sum_i x_i=0\}\subseteq \mathbb{R}^{n+2}$.

Take its boundary complex $\mathbb{S}^n$, subdivide once, project radially to $S^n$, and quotient by the antipodal map.

</details>
<details>
<summary>(c)</summary>

$H_n(S^n)\cong\mathbb{Z}$.

Since the antipodal map $a$ has no fixed points, $L(a)=0$. Then $\text{tr}(a_{\star n})=(-1)^{n+1}$, so $a_{\star n}=\text{Id}$ for $n$ odd, and $a_{\star n}=-\text{Id}$ for $n$ even.

</details>
<details>
<summary>(d)</summary>

Suppose $f$ exists. Then it is homotopic to the identity (by linearly interpolating and then projecting back to the sphere), so $L(f)=2$. But $f$ has no fixed points.$_\#$

</details>
